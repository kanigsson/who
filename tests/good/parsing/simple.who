let f (x : int) = {} x {}
