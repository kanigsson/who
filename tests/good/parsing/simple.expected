let f  = (Î»(x:int) -> x) in ()
